index.html

```diff
@@ -61,9 +60,152 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+                            <ul class="project-tech-stack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectfour.png">
+                            <h3 class="ptitle">Multi-Post Stories</h3>
+                        </div>
+                        <div class="p-details">
+                            <ul>
+                                <li>CANOPY</li>
+                                
+                                <li>Back End Dev</li>
+                               
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="lower_part">
+                        <ul class="project-tech-stack">
+                            <li><a>HTML</a></li>
+                            <li><a>CSS</a></li>
+                            <li><a>Javascript</a></li>
+                        </ul>
+                        <a class="button">See project</a>
+                        </div>
+                    </section>
+            </section>
+            
+            
+            <section class="about-me">
+                <h2 class="titletext">About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="letconnect"> LET’S CONNECT</p>
+                           
+                       
+
+                    
+                <div class="socialmedia">
+                    <ul class="media">                           
+                        
+                        <li><a class="media"><img src="img/media-3.png"></a></li>
+                        <li><a class="media"><img src="img/media-4.png"></a></li>
+                        <li><a class="media"><img src="img/media-1.png"></a></li>
+                        <li><a class="media"><img src="img/media-5.png"></a></li>
+                        <li><a class="media"><img src="img/media-2.png"></a></li>
+                    </ul>
+
+                </div>
+
+                <a class="button">Get my resume</a>
+
+                <div class="skills">
+                    <ul>
+                        <li>Languages</li>
+                        <li> <img src="img/Langguage.png"></li>
```

Comment:
:+1: Well done on showcasing the languages skill section! However, there is room for improvement to align it with the design:

- [ ]  Instead of using an image to represent the languages, it is important to code and display them as text, just as shown in the design. This will allow for better accessibility, searchability, and flexibility in the future when adding more languages. :pray:

By implementing these changes, you will bring the language skill section closer to the intended design, making it more effective and aligned with best practices. Keep up the great work!